### PR TITLE
select: make it work

### PR DESCRIPTION
**Pull request by dfilatov**

### CODE REVIEW

File: common.blocks/select/_type/select_type_radio.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ modules.define('select', function(provide, Select) {
  */
 provide(Select.decl({ modName : 'type', modVal : 'radio' }, /** @lends select.prototype */{
     _updateControl : function() {
-        this.elem('control').val(JSON.stringify(this.getVal()));
+        var val = this.getVal();
+        this.elem('control').val(typeof val === 'string'? val : JSON.stringify(val));
```
Comment (narqo) [inline common.blocks/select/_type/select_type_radio.js]: Maybe we could check `typeof val !== 'object'? val : JSON.strinigify(val)`? Guess jQuery would do the rest.

File: common.blocks/select/select.js [hunk 2]
```diff
@@ -230,7 +232,7 @@ provide(BEMDOM.decl(this.name, /** @lends select.prototype */{
             'type="hidden" ' +
             'name="' + name + '" ' +
             'class="' + this.buildClass('control') + '" ' +
-            'value="' + escape.attr(JSON.stringify(val)) + '"/>';
+            'value="' + escape.attr(typeof val === 'object'? JSON.stringify(val) : val) + '"/>';
```
Comment (narqo) [inline common.blocks/select/select.js:235]: :)

**Comment by narqo** [comment]
looks complicated but right :) :+1:

File: common.blocks/select/_type/select_type_radio.js (hunk 1 above)
Comment (dfilatov) [inline common.blocks/select/_type/select_type_radio.js]: fixed

**Pull request merged**